libcontainer/specconv/spec_linux.go

```diff
@@ -190,6 +190,23 @@ func CreateLibcontainerConfig(opts *CreateOpts) (*configs.Config, error) {
 
 	exists := false
 	for _, m := range spec.Mounts {
+		// check mount options for (b)bind type mount
+		if m.Type == "bind" || m.Type == "rbind" {
+			if len(m.Options) == 0 {
+				return nil, fmt.Errorf("no options specified for %v type mount: [destination=%v, source=%v].", m.Type, m.Destination, m.Source)
```

Comment:
suggest making this a warning "logrus.Warnf" vs throwing an error..

As Mr. Crosby said we can let the sys call throw the error.